app/code/pagetype/ContentPage.php

```diff
@@ -3,8 +3,44 @@
 namespace App\PageType;
 
 use Page;
+use SilverStripe\Forms\TextareaField;
 
 class ContentPage extends Page
 {
+    /**
+     * Page description in CMS
+     *
+     * @var string
+     */
     private static $description = "Content page";
+
+    /**
+     * Page database name
+     *
+     * @var string
+     */
+    private static $table_name = "ContentPage";
+
+    /**
+     * Page database
+     *
+     * @var array
+     */
+    private static $db = [
+        'Summary' => 'Text'
+    ];
+
+    /**
+     * Create fields in the page settings of the CMS
+     *
+     * @return void
+     */
+    public function getCMSFields()
+    {
+        $fields = parent::getCMSFields();
+        $fields->addFieldToTab('Root.Main', TextareaField::create(
```

Comment:
Have you got PSR-2 linting on?

With PSR-2 multi-line arguments for functions are formatted like this:

```$fields->addFieldToTab(
             'Root.Main', 
             TextareaField::create('Summary'),
             'Content'
        );
```

try reformat it in this way :)